tests/sentry/models/test_projecttemplateoption.py

```diff
@@ -0,0 +1,30 @@
+from sentry.models.options.project_template_option import ProjectTemplateOption
+from sentry.models.projecttemplate import ProjectTemplate
+from sentry.testutils.cases import TestCase
+
+
+class ProjectTemplateOptionTest(TestCase):
+    def setUp(self):
+        self.org = self.create_organization()
+
+        self.project_template = ProjectTemplate.objects.create(
+            name="test_project_template", organization=self.org
+        )
+
+    def tearDown(self):
```

Comment:
nit: what is the utility of deleting these from the test db? We create a new set for each test, and the delete operation is probably more expensive, running-time wise, than simply having more rows in these tables.

Reply:
https://github.com/getsentry/sentry/pull/69816/files#r1583973006

This is being added as a placeholder for additional testing. I wanted to make and setup the test infra structure while the model code is still very simple.